app/views/submissions/_submissions_display.html.erb

```diff
@@ -3,64 +3,73 @@
     <div class="col-lg-12">
       <h1 class="page-header"><%= project_level_header(project_level_text(locals[:selected_type]), locals[:teams].length) %></h1>
     </div>
-    <% locals[:teams].each_with_index do |team, idx| %>
+  </div>
+  <% teams, max_no_of_columns = locals[:teams], 4 %>
+  <% for i in (0..teams.length - 1).step(max_no_of_columns) do %>
+    <div class="row">
       <!-- Thumbnail display for each team -->
-      <div class="col-lg-3 col-md-4 col-xs-6 thumb">
-        <%= link_to image_tag(image_tag_string(locals[:selected_type]), class: 'img-rounded') %>
-        <h3>
-          <button type="button" class="btn button_team btn-lg" id="button_<%= idx %>" data-toggle="modal" data-target="#modal_<%= team.id %>">
-            <%= idx + 1 %>. <%= team.team_name %>
-          </button>
-        </h3>
-        <p class="text-muted">Students: <%= team.students.map {|student| student.user.user_name}.join(' & ') %></p>
-        <p class="text-muted">Advised by: <%= team.adviser ? team.adviser.user.user_name : 'Not assigned' %></p>
-      </div>
-      <!-- End of thumbnail display -->
+      <% for idx in (i ..i + max_no_of_columns - 1) do %>
+        <% team = teams[idx] %>
+        <% if team == nil %>
+          <% break %>
+        <% end %>
+        <div class="col-lg-3 col-md-4 col-xs-6 thumb">
```

Comment:
1) 4 columns each with width equal to 3 Bootstrap columns (Large devices)
2) 3 columns each with width equal to 4 Bootstrap columns (Medium devices)
3) 2 columns each with width equal to 6 Bootstrap columns (Small and Extra Small Devices)